pkg/policy/rule_test.go

```diff
@@ -638,6 +639,36 @@ func (ds *PolicyTestSuite) TestL3Policy(c *C) {
 	c.Assert(err, Not(IsNil))
 }
 
+func (ds *PolicyTestSuite) TestL3PolicyRestrictions(c *C) {
+	// Check rejection of allow-all CIDRs
+	barSelector := api.NewESFromLabels(labels.ParseSelectLabel("bar"))
+	cidrs := []api.CIDR{"0.0.0.0/0"}
+	apiRule1 := api.Rule{
+		EndpointSelector: barSelector,
+		Ingress:          []api.IngressRule{{FromCIDR: cidrs}},
+	}
+	err := apiRule1.Sanitize()
+	c.Assert(err, Not(IsNil))
```

Comment:
I feel a bit uneasy about this behaviour but I understand given the current implementation.